src/classes/ADDR_Account_TDTM.cls

```diff
@@ -36,6 +36,9 @@
 */
```

Comment:
Classes and triggers should use the `UpperCamelCase` naming convention. I suggest to rename accordingly.